frontend/packages/topology/src/behavior/withCreateConnector.tsx

```diff
@@ -135,6 +135,22 @@ const CreateConnectorWidget: React.FC<CreateConnectorWidgetProps> = observer((pr
     );
   }
 
+  const unsetHandle = React.useRef<number>();
+
+  React.useEffect(() => {
+    setHover(false);
+    clearTimeout(unsetHandle.current);
+    unsetHandle.current = window.setTimeout(() => setHover(true), 2000);
+    return () => {
+      clearTimeout(unsetHandle.current);
+    };
+  }, [endPoint.x, endPoint.y]);
```

Comment:
This is better off going directly into the `DefaultCreateConnector`.

In dev-console we can supply our own prop to `renderConnector` which uses `DefaultCreateConnector` to supply the custom message.